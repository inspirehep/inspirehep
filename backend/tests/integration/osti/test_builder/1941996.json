{
    "core":true,
    "dois":[
       {
          "value":"10.2172/1827835"
       }
    ],
    "self":{
       "$ref":"https://inspirehep.net/api/literature/1941996"
    },
    "urls":[
       {
          "value":"https://lss.fnal.gov/archive/2021/slides/fermilab-slides-21-034-t.pdf",
          "description":"Fermilab Library Server"
       }
    ],
    "titles":[
       {
          "title":"Lattice QCD and Î½A",
          "source":"submitter"
       }
    ],
    "$schema":"https://inspirehep.net/schemas/records/hep.json",
    "authors":[
       {
          "ids":[
             {
                "value":"M.L.Wagman.1",
                "schema":"INSPIRE BAI"
             }
          ],
          "uuid":"6a11abf8-19b1-4c48-9b4d-0f8e7ccab9c7",
          "record":{
             "$ref":"https://inspirehep.net/api/authors/1501833"
          },
          "full_name":"Wagman, Michael L.",
          "affiliations":[
             {
                "value":"Fermilab",
                "record":{
                   "$ref":"https://inspirehep.net/api/institutions/902796"
                }
             }
          ],
          "signature_block":"WAGNANm"
       }
    ],
    "curated":true,
    "keywords":[
       {
          "value":"lattice field theory",
          "schema":"INSPIRE",
          "source":"classifier"
       }
    ],
    "documents":[
       {
          "key":"cf213fc7303e164c6c4ec6d69af22223",
          "url":"https://inspirehep.net/files/cf213fc7303e164c6c4ec6d69af22223",
          "filename":"fermilab-slides-21-034-t.pdf",
          "fulltext":true,
          "description":"Fulltext",
          "original_url":"https://lss.fnal.gov/archive/2021/slides/fermilab-slides-21-034-t.pdf"
       }
    ],
    "_collections":[
       "Fermilab"
    ],
    "document_type":[
       "article"
    ],
    "control_number":1941996,
    "report_numbers":[
       {
          "value":"FERMILAB-SLIDES-21-034-T",
          "source":"submitter"
       }
    ],
    "publication_info":[
       {
          "cnum":"C21-03-15.6",
          "conference_record":{
             "$ref":"https://inspirehep.net/api/conferences/1941930"
          }
       }
    ],
    "acquisition_source":{
       "email":"jtrombet@fnal.gov",
       "orcid":"0000-0002-9381-2678",
       "method":"submitter",
       "source":"submitter",
       "datetime":"2021-10-12T16:44:20.523867",
       "internal_uid":73552,
       "submission_number":"3410255"
    },
    "inspire_categories":[
       {
          "term":"Lattice"
       }
    ],
    "external_system_identifiers":[
       {
          "value":"1827835",
          "schema":"OSTI"
       }
    ]
 }