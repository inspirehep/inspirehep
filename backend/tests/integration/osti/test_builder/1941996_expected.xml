<?xml version="1.0" encoding="UTF-8"?>
<records>
   <record>
      <new />
      <site_input_code>FNAL</site_input_code>
      <product_type product_subtype="R">CO</product_type>
      <access_limitation>
         <unl />
      </access_limitation>
      <site_url>https://lss.fnal.gov/archive/2021/slides/fermilab-slides-21-034-t.pdf</site_url>
      <title>Lattice QCD and Î½A</title>
      <authors>
         <authors_detail>
            <first_name>Michael</first_name>
            <middle_name>L.</middle_name>
            <last_name>Wagman</last_name>
            <affiliation>Fermilab</affiliation>
            <private_email />
            <orcid_id />
         </authors_detail>
      </authors>
      <contributor_organizations />
      <report_nos>FERMILAB-SLIDES-21-034-T</report_nos>
      <doe_contract_nos>AC02-07CH11359</doe_contract_nos>
      <site_input_code>FNAL</site_input_code>
      <country_publication_code>US</country_publication_code>
      <sponsor_org>USDOE Office of Science (SC), High Energy Physics (HEP) (SC-25)</sponsor_org>
      <released_by>Heath O'Connell</released_by>
      <released_by_email>hoc@fnal.gov</released_by_email>
      <released_by_phone>630-840-6017</released_by_phone>
      <medium_code>ED</medium_code>
      <file_format>PDF/A</file_format>
      <description />
      <originating_research_org>Fermi National Accelerator Laboratory (FNAL), Batavia, IL (United States)</originating_research_org>
      <journal_name />
      <journal_volume />
      <journal_issue />
      <product_size />
      <doi>10.2172/1827835</doi>
      <conference_information> 03/15-03/18</conference_information>
      <other_identifying_nos>oai:inspirehep.net:1941996</other_identifying_nos>
      <publication_date>1900</publication_date>
      <language>English</language>
      <subject_category_code />
      <released_date>10/28/2021</released_date>
   </record>
</records>